src/models/backbones/swin.py

```diff
@@ -0,0 +1,451 @@
+"""TorchOK Swin Transformer V2
+
+Adapted from https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/swin_transformer_v2.py
```

Comment:
How much is actually adpoted from timm? If the weights only, then I suggest to put this note above the weights configuration block

Reply:
Keep in mind that timm has a different license than Microsoft's repo